app/javascript/App/index.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react'
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
+import { MuiThemeProvider } from '@material-ui/core/styles'
+import theme from '~/theme'
+
+import ExchangeListingsPage from '~/bundles/ExchangeListings/ExchangeListingsCointainer'
+import NewsfeedPageNew from '~/bundles/NewsfeedPage/NewsfeedPageContainer'
+import CoinIndex from './components/CoinIndex'
+import CoinListContainer from '~/bundles/common/containers/CoinListContainer'
+import NewsfeedContainer from '~/bundles/NewsfeedPage/NewsfeedContainer'
+import ReactOnRails from 'react-on-rails'
+
+const App = (props, railsContext) => (
```

Comment:
This file now contains only main app with react router. All other components are imported in `startup` file to make the structure cleaner.

Reply:
Thats much cleaner, thanks